app/features/app/components/App.js

```diff
@@ -3,7 +3,9 @@
 import { AtlasKitThemeProvider } from '@atlaskit/theme';
 
 import React, { Component } from 'react';
-import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import { Route, Switch } from 'react-router';
+import { ConnectedRouter as Router } from 'react-router-redux';
```

Comment:
unwritten style rule: group these imports and sort them alphabetically.

also, relative imports are sorted from "furthest" to "closest"

Reply:
Okay, I moved history import wih other relative imports.
So its looks like this now.
```
import { AtlasKitThemeProvider } from '@atlaskit/theme';

import React, { Component } from 'react';
import { Route, Switch } from 'react-router';
import { ConnectedRouter as Router } from 'react-router-redux';

import { Conference } from '../../conference';
import config from '../../config';
import { history } from '../../router';
import { Welcome } from '../../welcome';
```
Questions
- Do you want me to group relative imports with react and router imports too?
- Do we sort module name or component name alphabeltically? ( `react-router` should come above `react-redux-router` or `ConnectedRouter` should come above `Router, Switch`)